content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -294,6 +294,11 @@ command line arguments to `kube-apiserver`:
 * `--service-account-signing-key-file`
 * `--api-audiences` (can be omitted)
 
+You can specify the `--service-account-issuer` argument multiple times. This can be useful to
+enable a non-disruptive change of the key for an issuer.
+You must be running running Kubernetes v1.22 or later to be able to specify
+`--service-account-issuer` multiple times.
```

Comment:
This isn't _wrong_ but it's also not at all clear (to a reader) what the deal is about specifying the other arguments such as `--service-account-key-file` multiple times.

Readers have to go and look at https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/ to work it out.
It would be nice to save them some of that effort.

Reply:
Hi @sftim, can we write like this so that Readers have some info regarding these flags? Please take a look and let me know your thoughts on this. if it seems good then I will add this.
```
`--service-account-issuer`  can use for the Identifier of the service account token issuer. The issuer will assert this identifier in the "iss" claim of issued tokens. This value must be a string or URI.

You can specify specified `--service-account-key-file` flag multiple times with different files, It must be specified when 
`--service-account-signing-key` is provided. This File contains PEM-encoded x509 RSA or ECDSA private or public keys which are used to verify ServiceAccount tokens. No need for k8s version v1.22 or greater, it also works in the lower version of k8s v1.22.

You can use `--service-account-signing-key-file` flag for the path to the file that contains the current private key of the service account token issuer. The issuer will sign issued ID tokens with this private key.

You can use `--api-audiences ` flag for Identifiers of the API. We can specify multiple audiences for whom the token is used. You can omit it also if not specified then it defaults to a single element list containing the issuer URL.

For more information please see various flags used in [kube-apiserver](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/).
```

Reply:
The behavior when the flags are specified times should be described. If `service-account-key-file` or `api-audiences` is specified multiple times, tokens signed by any of the specified keys or for any of the specified audiences are considered valid by the Kubernetes API server.

Reply:
Hi @liggitt Thanks for looking at this.


> The behavior when the flags are specified times should be described. If `service-account-key-file` or `api-audiences` is specified multiple times, tokens signed by any of the specified keys or for any of the specified audiences are considered valid by the Kubernetes API server.

Sure, I will add this thing to it.

Reply:
Hi, @liggitt @sftim  I modified it acc. to the suggestion. we can append these lines to the commit. Please take a look at this one,  and let me know any kind of changes or modifications you want.

```
`--service-account-issuer`  can use for the Identifier of the service account token issuer. The issuer will assert this identifier in the "iss" claim of issued tokens. This value must be a string or URI. If `service-account-issuer` is specified multiple times, tokens issued by any of the specified issuers are considered valid by the Kubernetes API server.

You can specify specified `--service-account-key-file` flag multiple times with different files, It must be specified when 
`--service-account-signing-key` is provided. This File contains PEM-encoded x509 RSA or ECDSA private or public keys which are used to verify ServiceAccount tokens. No need for k8s version v1.22 or greater, it also works in the lower version of k8s v1.22. If `service-account-key-file` is specified multiple times, tokens signed by any of the specified keys are considered valid by the Kubernetes API server.

You can use `--service-account-signing-key-file` flag for the path to the file that contains the current private key of the service account token issuer. The issuer will sign issued ID tokens with this private key.

You can use `--api-audiences ` flag for Identifiers of the API. We can specify multiple audiences for whom the token is used. You can omit it also if not specified then it defaults to a single element list containing the issuer URL. If `api-audiences` is specified multiple times, tokens for any of the specified audiences are considered valid by the Kubernetes API server.

For more information please see various flags used in [kube-apiserver](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/).
```

Thanks.

Reply:
Hi @sftim @liggitt  
Could you please take a look? and  let me know your thoughts on this.

> Hi, @liggitt @sftim I modified it acc. to the suggestion. we can append these lines to the commit. Please take a look at this one, and let me know any kind of changes or modifications you want.
> 
> ```
> `--service-account-issuer`  can use for the Identifier of the service account token issuer. The issuer will assert this identifier in the "iss" claim of issued tokens. This value must be a string or URI. If `service-account-issuer` is specified multiple times, tokens issued by any of the specified issuers are considered valid by the Kubernetes API server.
> 
> You can specify specified `--service-account-key-file` flag multiple times with different files, It must be specified when 
> `--service-account-signing-key` is provided. This File contains PEM-encoded x509 RSA or ECDSA private or public keys which are used to verify ServiceAccount tokens. No need for k8s version v1.22 or greater, it also works in the lower version of k8s v1.22. If `service-account-key-file` is specified multiple times, tokens signed by any of the specified keys are considered valid by the Kubernetes API server.
> 
> You can use `--service-account-signing-key-file` flag for the path to the file that contains the current private key of the service account token issuer. The issuer will sign issued ID tokens with this private key.
> 
> You can use `--api-audiences ` flag for Identifiers of the API. We can specify multiple audiences for whom the token is used. You can omit it also if not specified then it defaults to a single element list containing the issuer URL. If `api-audiences` is specified multiple times, tokens for any of the specified audiences are considered valid by the Kubernetes API server.
> 
> For more information please see various flags used in [kube-apiserver](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/).
> ```
> 
> Thanks.